### DISCUSSION TITLE
Make it harder to enter the settings page

### DISCUSSION THREAD

**Discussion by l4n1skyy** [jordwest/news-feed-eradicator#431] [Ideas]
This is just a suggestion, but I feel like the settings page link and the toggle next to the quote isn't such a good idea. We can easily access the settings page by just clicking on the extension, but at least it's hidden a little in our extension list. These additional features make it really tempting to go to the settings page and just switch the extension off. Thanks for reading!

**Comment by mitar** [comment]
Related: #214 #193.

**Comment by mitar** [comment]
I suggest adding a simple delay instead: #258

**Comment by gregorylearns** [comment]
Why not both? But I agree that the link to the settings page shouldn't be easily accessible. It becomes second nature for me to bypass the eradication
